<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>931</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page0">
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>104</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="txtLable">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Txt文件操作</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>861</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="searchEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>691</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>数据库操作</string>
      </property>
      <widget class="QPushButton" name="dataSearch">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/image/res/search.png</normaloff>:/image/res/search.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="dataSave">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/image/res/save.png</normaloff>:/image/res/save.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="dataEmpty">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/image/res/qingkong.png</normaloff>:/image/res/qingkong.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="dataAdd">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/image/res/add.png</normaloff>:/image/res/add.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="dataDelete">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/image/res/delete.png</normaloff>:/image/res/delete.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox_Chose">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>11</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>产品类型</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>产品编号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>装配状态</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>完成时间</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>221</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>开发者选项</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_aboutproduct"/>
   </widget>
   <widget class="QMenu" name="ExploreOption">
    <property name="title">
     <string>开发者选项</string>
    </property>
    <addaction name="action_enter"/>
   </widget>
   <addaction name="File"/>
   <addaction name="ExploreOption"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_init"/>
   <addaction name="separator"/>
   <addaction name="action_newprj"/>
   <addaction name="separator"/>
   <addaction name="action_circleMode"/>
   <addaction name="separator"/>
   <addaction name="action_start"/>
   <addaction name="separator"/>
   <addaction name="action_stop"/>
   <addaction name="separator"/>
   <addaction name="action_reset"/>
   <addaction name="separator"/>
   <addaction name="action_database"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/file-open.png</normaloff>:/image/res/file-open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开文件(O）</string>
   </property>
  </action>
  <action name="action_newprj">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="创建任务" resource="res.qrc">
     <normaloff>:/image/res/doc-new.png</normaloff>:/image/res/doc-new.png</iconset>
   </property>
   <property name="text">
    <string>创建任务</string>
   </property>
   <property name="toolTip">
    <string>创建任务</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/file-save.png</normaloff>:/image/res/file-save.png</iconset>
   </property>
   <property name="text">
    <string>保存文件(&amp;S) </string>
   </property>
   <property name="toolTip">
    <string>保存文件(S) </string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/doc-quit.png</normaloff>:/image/res/doc-quit.png</iconset>
   </property>
   <property name="text">
    <string>关闭退出(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>关闭退出(C)</string>
   </property>
  </action>
  <action name="action_aboutproduct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/help.png</normaloff>:/image/res/help.png</iconset>
   </property>
   <property name="text">
    <string>关于产品</string>
   </property>
  </action>
  <action name="action_init">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/doc-init.png</normaloff>:/image/res/doc-init.png</iconset>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
   <property name="toolTip">
    <string>初始化</string>
   </property>
  </action>
  <action name="action_transformerassembly">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/doc-measure.png</normaloff>:/image/res/doc-measure.png</iconset>
   </property>
   <property name="text">
    <string>变压器装配</string>
   </property>
   <property name="toolTip">
    <string>尺寸测量</string>
   </property>
  </action>
  <action name="action_ex-process">
   <property name="text">
    <string>图像预处理</string>
   </property>
   <property name="toolTip">
    <string>图像预处理</string>
   </property>
  </action>
  <action name="action_trans_2">
   <property name="text">
    <string>坐标变换</string>
   </property>
   <property name="toolTip">
    <string>坐标变换</string>
   </property>
  </action>
  <action name="action_mea">
   <property name="text">
    <string>尺寸测量</string>
   </property>
   <property name="toolTip">
    <string>尺寸测量</string>
   </property>
  </action>
  <action name="action_juade">
   <property name="text">
    <string>装配状态判断</string>
   </property>
   <property name="toolTip">
    <string>装配状态判断</string>
   </property>
  </action>
  <action name="action_match">
   <property name="text">
    <string>模板匹配</string>
   </property>
   <property name="toolTip">
    <string>模板匹配</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/reset.png</normaloff>:/image/res/reset.png</iconset>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
   <property name="toolTip">
    <string>复位</string>
   </property>
  </action>
  <action name="action_start">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/doc-start.png</normaloff>:/image/res/doc-start.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="toolTip">
    <string>开始</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/stop.png</normaloff>:/image/res/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>stop</string>
   </property>
  </action>
  <action name="action_database">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/doc-seach.png</normaloff>:/image/res/doc-seach.png</iconset>
   </property>
   <property name="text">
    <string>数据库</string>
   </property>
   <property name="toolTip">
    <string>数据库</string>
   </property>
  </action>
  <action name="action_white">
   <property name="text">
    <string>白色风格</string>
   </property>
   <property name="toolTip">
    <string>白色风格</string>
   </property>
  </action>
  <action name="action_black">
   <property name="text">
    <string>黑色风格</string>
   </property>
   <property name="toolTip">
    <string>黑色风格</string>
   </property>
  </action>
  <action name="action_gray">
   <property name="text">
    <string>灰色风格</string>
   </property>
   <property name="toolTip">
    <string>灰色风格</string>
   </property>
  </action>
  <action name="action_dataconnect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/link.png</normaloff>:/image/res/link.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>连接</string>
   </property>
  </action>
  <action name="action_databreak">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/break.png</normaloff>:/image/res/break.png</iconset>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
   <property name="toolTip">
    <string>断开</string>
   </property>
  </action>
  <action name="action_enter">
   <property name="text">
    <string>进入开发者模式</string>
   </property>
   <property name="toolTip">
    <string>进入开发者模式</string>
   </property>
  </action>
  <action name="action_circleMode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/kaishi.png</normaloff>:/image/res/kaishi.png</iconset>
   </property>
   <property name="text">
    <string>循环执行</string>
   </property>
   <property name="toolTip">
    <string>循环执行</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
